doc/tips.rst

```diff
@@ -16,15 +16,27 @@ reproduce an umpf!
 Use ``--identical`` only for specific branches (but use tip-of-head for others)
 -------------------------------------------------------------------------------
 
-Currently, ``--identical`` will always use the hashinfo for all branches.
+By default, ``--identical`` will always use the hashinfo for all branches.
+This can be overridden case by case with the ``--override`` option.
+For example, to use the local state of only the ``v4.19/topic/imx-poweroff``
+branch, but use ``acme`` remote for everything else::
 
-Workround 1: reset all local topic branches that are included in the umpf to their
-desired commit, then let umpf prompt for the remote for every single branch
-(see above), and select the local branch.
+  umpf --identical --remote=acme --override v4.19/topic/imx-poweroff build series
 
-Workaround 2: If the same state is needed on future umpfs (e.g. with stable workflows),
-create a separate branch in the ``vX.Y/customers/`` namespace on the desired commit,
-which you can then refer to in your *useries* file or with ``umpf merge``.
+In cases, where the local branch is named differently, this can be made
+explicit::
+
+  umpf --identical --remote=acme --override \
+    v4.19/topic/imx-poweroff:v4.19/topic/poweroff-rework build series
+
+Normally, this state is needed for future umpfs and thus ``--override`` should
+be a temporary measure until the remote branches are updated and future
+umpfs yield the same result without ``--override``.
```

Comment:
The way you implement this, `--overwrite` works for non-identical umpfs as well. That should be documented.

Reply:
This is the trips and tricks section, which I don't think is meant to be exhaustive. The option itself is documented normally in the help text without reference to being limited to `--identical`.